packages/Algernon.package/ALGWorldMenuType.class/class/menuEntries.st

```diff
@@ -5,11 +5,11 @@ menuEntries
 	entries := OrderedCollection new.	
 
 	((self menus
-		collect: [ :each | each subMenu ])
-		reject: [ :each | each isNil ])
+		collect: [ :each | {each subMenu. each contents} ])
+		reject: [ :each | each first isNil ])
 		collect: [ :each |
```

Comment:
I know that the original implementation came from another issue, but do we really collect sth here? As we just add entries to an outer local variable, a do instead of collect would have the same effect right?

Reply:
You're right, absolutely nothing is collected here, an luckily we never touched this collected nothing🤣

I just feel like `extractMenuEntriesFrom:` is still not quite the right name, since the extracted menu entries may contain line spacers or some other bloat, which won't be filtered out until the very end of the `menuEntries` function in `entries select: [:each | each isKindOf: MenuItemMorph]`

Reply:
Ahh, i didn't note that the line spacers and other bloat won't be filtered out until later. Then Niklas' "scrape" suggestion from the other PR should fit better. Sth like `scrapeMenuItemsFrom` as "MenuItems" includes those bloat items more then "MenuEntries" and scraping underlines that there is bloat included?

Reply:
@niklasmohrin Thoughts?

Reply:
Yes I have foreseen this miles away and am very pleased that you have finally given in /s

Yeah, that all sounds right to me 👍